pkgs/tools/networking/dnscrypt-proxy/2.x/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, go }:
```

Comment:
I don't think you need go here

Reply:
Okay, I'm trying to rebuild without it but removing it doesn't seem to invalidate the already-built binary.